src/i18n-cache/.gitignore

```diff
@@ -1,4 +0,0 @@
-# ignore all the contents of this folder except .gitignore and index.js
-*
-!.gitignore
-!index.js
```

Comment:
This file is no longer required as it was removed in https://github.com/wordpress-mobile/gutenberg-mobile/pull/4411/files?show-deleted-files=true&show-viewed-files=true&file-filters%5B%5D=#diff-87aa833f614fbab0d0325cfe7cf2cf7f3f998a9c60e03bd9109e259859f360bf.